package.json

```diff
@@ -33,13 +33,13 @@
   "main": "./lib",
   "scripts": {
     "build": "node bin/build.js",
-    "lint": "npm run lint:src && npm run lint:lockfile && npm run lint:markdown",
+    "lint": "npm-run-all --parallel lint:*",
     "lint:lockfile": "lockfile-lint --allowed-hosts npm --allowed-schemes https: --empty-hostname false --type npm --path package-lock.json",
     "lint:markdown": "markdownlint \"**/*.md\" --ignore node_modules --config .markdownlint.js",
     "lint:src": "eslint --cache --cache-location node_modules/.cache/.eslintcache --report-unused-disable-directives .",
     "nyc": "nyc",
     "mocha": "mocha test test/package-managers/npm test/package-managers/yarn && mocha --exit test/timeout",
-    "test": "npm run lint:all && npm run mocha",
```

Comment:
This was broken BTW since there was no `lint:all` script

Reply:
Oops :)